### PR TITLE
Update videos and make links visible on homepage

### PR DESCRIPTION
**Pull request by mraible**
Here's what the homepage looks like after adjusting CSS to make links visible.

Screen Shot 2019-09-20 at 11 26 32 AM

Screen Shot 2019-09-20 at 11 26 42 AM

### CODE REVIEW

**Comment by mraible** [comment]
@jdubois If these changes are OK with you, can you please merge? Shortly after merging, I want to encourage other JHipster contributors to add their videos b/c there's likely more from 2019.

**Comment by jdubois** [comment]
Yes of course, I just merged it.
I'm trying to have a list of JHipster videos at https://www.youtube.com/playlist?list=PLCJhCeRfMgNKITROqCz3GuRlKOx1f3S3p but it's hard to follow (which is awesome, in fact!)
